waitress/runner.py

```diff
@@ -171,6 +171,18 @@ def show_help(stream, name, error=None): # pragma: no cover
         print('Error: {0}\n'.format(error), file=stream)
     print(HELP.format(name), file=stream)
 
+
+def show_exception(stream): # pragma: no cover
```

Comment:
That is true.  I blindly followed the print-help function.  I'll add some tests.